readme.md

```diff
@@ -0,0 +1,130 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# House Frontend
```

Comment:
- [ ] There are 2 readme files on your repo.  In order to prevent confusion for your users, and Github, kindly remove 1 of them.  👍